Sources/SentryCrash/Recording/Tools/SentryCrashJSONCodecObjC.m

```diff
@@ -345,15 +345,28 @@ - (void)dealloc
     }
 
     if ([object isKindOfClass:[NSDictionary class]]) {
+        NSDictionary *dict = (NSDictionary *)object;
         if ((result = sentrycrashjson_beginObject(context, cName)) != SentryCrashJSON_OK) {
             return result;
         }
-        NSArray *keys = [(NSDictionary *)object allKeys];
-        if (codec->_sorted) {
+        NSArray *keys = [dict allKeys];
+
+        bool allKeysOfSameType = true;
```

Comment:
L: I've seen most of the time booleans using `YES/NO`, should we stick to it, or is there a reason to use `true/false`? no strong opinions here.